src/main/resources/view/ListPanelStackPane.fxml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.layout.StackPane?>
+
+
+<StackPane xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
+</StackPane>
```

Comment:
This fxml file isn't being used anywhere, I'm quite sure you can remove it safely.